debug_validation.py

```
@@ -47,17 +88,20 @@ def debug_validation():
         print(f"📡 Expected 400: {response.status_code == 400}")
         try:
             result = response.json()
-            print(f"📊 Response JSON: {result}")
+            sanitized_result = sanitize_for_logging(str(result))
+            print(f"📊 Response JSON: {sanitized_result}")
         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)